rasa/telemetry.py

```diff
@@ -717,6 +722,8 @@ async def track_model_training(
             # Old nomenclature from when 'responses' were still called
             # 'templates' in the domain
             "num_templates": len(domain.responses),
+            "num_conditional_response_variations": count_conditional_responses,
+            "has_conditional_response_variations": has_conditional_responses,
```

Comment:
If we are also recording counts of CRVs do we need to still record this boolean field?

Reply:
Very good point, @wochinge asked the same - I can remove the boolean field, I just thought both metrics are wanted based on what was agreed in the scoping doc.